Documentation.md

```diff
@@ -0,0 +1,207 @@
+Welcome! You can find here documentation for /r/oneplus Discord server bot.
+
+Its prefix is `;` but commands can also be triggered by pinging bot.\
+Note that commands only works in guilds.
+
+# Features:
+
+## Owner
+
+### Emergency bot shutdown
+
+`;x`\
+Terminates bot process.
+
+### SQL commands
+
+`;sql [insert SQL command]`\
+Allow owner to edit content of SQL databases directly via a Discord message.
+
+### Update
+
+Updates the database with the current state of the server.
+
+## Administration
+
+### Modmail
+
+There exists a category (configurable in database) with the purpose of modmail. When a user dms the bot, a channel in this category the modmail channels will be created. There also exists another separate channel for modmaillog, in which the messages will get stored when the thread has been closed.\
+The user will get a response, that the inquiry is being handled and moderators get a notification in modqueue.\
```

Comment:
Add that the staff role gets pinged (was not present when writing the PR this is based on)